src/mgmt/srv_drpc.c

```diff
@@ -259,6 +260,11 @@ create_pool_props(daos_prop_t **out_prop, char *owner, char *owner_grp,
 		idx++;
 	}
 
+	/* pool tiering policy */
+	new_prop->dpp_entries[idx].dpe_type = DAOS_PROP_PO_POLICY;
+	new_prop->dpp_entries[idx].dpe_val = policy;
```

Comment:
Just keep in mind that we may need to support more configurable values for each policy, like io sizes for SCM or performance tier or capacity tier, etc? (It's not necessary being implemented in this patch)